content/en/docs/Tracing/_index.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Tracing
```

Comment:
Can we put this under the Monitoring section rather than creating a new section just for it?

Reply:
Can we keep it separate for now ?

Reply:
Any compelling reason? Tracing is a function of monitoring.

Reply:
I usually consider them different things.